### PR TITLE
#170947493 User can  reset their password

### PR DESCRIPTION
**Pull request by Baraka-Mugisha**
#### What does this PR do?
- add reset password feature
#### Description of Task to be completed?
- added component for the feature
- added actions and reducers for the feature
- added storybook stories for the feature
- added tests for the feature

#### How should this be manually tested?
- clone repo
- run `npm run start:dev`
- use the following URL: `http://localhost:3000/password/forgot`
- provide valid email to which instructions to reset the pasword will be sent

#### What are the relevant pivotal tracker stories?
- [#170947493](https://www.pivotaltracker.com/story/show/170947493)
#### Screenshots (if appropriate)
forgot
reset

#### Questions:

### CODE REVIEW

File: src/redux/reducers/index.js [hunk 1]
```diff
@@ -4,3 +4,6 @@ import authReducer from './auth';
 export default combineReducers({
   user: authReducer,
 });
+
+// user: authReducer,
+// import authReducer from './authReducer';
```
Comment (bbaime98) [inline src/redux/reducers/index.js]: Remove this.

File: stories/2-Authentication.stories.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+export default {
+  title: 'Authentication',
+  component: resetPassword,
+};
+
+export const Default = () => <resetPassword />;
+
+// reset.story = {
+//   name: 'Reset Password',
```
Comment (bbaime98) [inline stories/2-Authentication.stories.js]: not needed as well.

**Review by bbaime98 — CHANGES_REQUESTED** [review]

File: src/redux/reducers/index.js (hunk 1 above)
Comment (Baraka-Mugisha) [reply src/redux/reducers/index.js]: I have removed them.

File: stories/2-Authentication.stories.js (hunk 2 above)
Comment (Baraka-Mugisha) [reply stories/2-Authentication.stories.js]: I have removed the file too

**Pull request closed without merging**